appveyor.yml

```diff
@@ -27,30 +29,50 @@ environment:
   matrix:
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: ""
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DEVENT__LIBRARY_TYPE=STATIC"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "autotools"
       EVENT_CONFIGURE_OPTIONS: ""
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "autotools"
       EVENT_CONFIGURE_OPTIONS: "--disable-openssl"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "autotools"
       EVENT_CONFIGURE_OPTIONS: "--disable-thread-support"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "autotools"
       EVENT_CONFIGURE_OPTIONS: "--disable-debug-mode"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "autotools"
       EVENT_CONFIGURE_OPTIONS: "--disable-malloc-replacement"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DEVENT__DISABLE_OPENSSL=ON"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DEVENT__DISABLE_THREAD_SUPPORT=ON"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DEVENT__DISABLE_DEBUG_MODE=ON"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DEVENT__DISABLE_MM_REPLACEMENT=ON"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
     - EVENT_BUILD_METHOD: "cmake"
       EVENT_CMAKE_OPTIONS: "-DCMAKE_C_FLAGS='-DUNICODE -D_UNICODE'"
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2017"
+    - EVENT_BUILD_METHOD: "cmake"
+      EVENT_CMAKE_OPTIONS: ""
+      APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2019"
```

Comment:
There are some [issues](https://ci.appveyor.com/project/libevent/libevent/builds/28226580/job/gi9jrofmlhfjr48j#L1137) with this build, can you take a look?

```
  WARN C:\projects\libevent\test\regress_ssl.c:210: Version mismatch for openssl: compiled with 1000214f but running with 1000212fOK
```

Reply:
I can't open this link: [issues](https://ci.appveyor.com/project/libevent/libevent/builds/28226580/job/gi9jrofmlhfjr48j#L1137), It says `Build not found or access denied.` And I can't reproduce the issue. Could you provide another link?

Reply:
That was failed build (due to flacky tests), and then I restart it and seems that in this case the build is unavailable after this (since I also got this error).

The reason why you do not see this errors, because ctest shows output only on failure, and the test do not fails now

Reply:
I'm going to merge this PR, but can you please take a look into this error? (I think that this can be related not only for VS2019)